### ISSUE TITLE
[QUESTION] Can I use the same images for distillation and fine-tuning?

### ISSUE THREAD

**Issue by enescicek** [lightly-ai/lightly-train#922]
I am planning to perform DINOv3 → YOLO distillation for segmentation.

I have 2,000 fully labeled images and will split them into:

- 1,600 training images
- 200 validation images
- 200 test images

Can I first use the 1,600 training images without their labels during the distillation stage, and then use the same 1,600 images with their labels during supervised fine-tuning?

Is this the recommended approach, or should the distillation and fine-tuning datasets be different?

**Comment by liopeer** [comment]
I would actually highly recommend using the same data (or at least data from the exact same distribution) for distillation and supervised fine-tuning. That usually gives the best results, since:
1. During distillation your backbone learns incredibly robust features;
2. During fine-tuning it's mainly the head that needs to learn the downstream task and adjust to the backbone.
The distillation process can help make the backbone less prone to overfitting. It's recommended to also play around with lower learning rates for the backbone during fine-tuning (lower than the head), if you have control over that pipeline.

**Comment by liopeer** [comment]
Closing this, since I think it has been answered. Feel free to reopen otherwise.
